style.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
 
 body {
   top: 0 !important;
```

Comment:
- [ ] OPTIONAL Please, consider removing `!important`, because important is a bad CSS practice. It disrupts the natural flow in applying the CSS rules where properties are applied from top to bottom. With `!important`, the property will now give priority to the latest important value 👍